### DISCUSSION TITLE
Bigger search component with clear

### DISCUSSION THREAD

**Discussion by Khushli** [twilio-labs/paste#1667] [Office hours] [answered]
I'm designing an internal information search tool which is basically google search for twilions. The project is in design and test phase, we need to get the mvp out in Q3.

- I need input on how to add a clear search functionality within the search bar. I found an earlier similar discussion [here](https://github.com/twilio-labs/paste/discussions/605#discussion-10993) but I don't see any updates on it in Paste yet.
- Also, wanted to explore if the search bar can change sizes based on where it's used in the UI.
- I'm using the combobox component 

(Have booked a OH for this)

**Comment by gloriliale** [comment]
Thanks for posting this here, @Khushli! We'll discuss during Office Hours and then post the recording below for reference. See ya then!

**Comment by andioneto** [comment, marked as answer]
Hi @Khushli 👋

Here is the [recording](https://www.loom.com/share/1774524827f2414ca9f704deb837e26d) from today's office hours, in case you'd like to revisit any of our conversation. You will find timestamps for each topic, and the password is posted in the #help-design-system channel.

To recap our discussion:
1. We discussed using element composition to create page hierarchy (e.g. input labels, or headers) rather than increasing the size of the combobox search input.
2. We reviewed a few different options for exposing a "clear" button on a search bar. The most current examples can be found in [this codesandbox](https://codesandbox.io/s/search-box-additional-examples-1g5gc?file=/src/index.tsx).

 Thanks again for coming to office hours!
